README.md

```diff
@@ -1,5 +1,8 @@
 # Overview
 
+[![Docker Repository on Quay](https://quay.io/repository/redhat-aqe/tox/status
```

Comment:
Container repository :-)

Reply:
I think it should be "Container image repository on Quauy" ;)

Reply:
oh, common! :D

Reply:
Quay.io or just Quay - but not Quauy :-)